src/Features/PlayerTrace.cpp

```diff
@@ -20,9 +20,13 @@ PlayerTrace* playerTrace;
 #define ADD_LINE_OVERLAY(...) engine->AddLineOverlay(nullptr, __VA_ARGS__)
 #endif
 
-Variable sar_player_trace_draw("sar_player_trace_draw", "0", "Display the recorded player trace\n");
 Variable sar_player_trace_record("sar_player_trace_record", "0", "Record player trace while set to 1\n");
 Variable sar_player_trace_size("sar_player_trace_size", "0", 0, "Maximum size of the player trace (in ticks), 0 for unlimited\n");
+Variable sar_player_trace_autoclear("sar_player_trace_autoclear", "1", "Automatically clear the trace on session start\n");
+
+Variable sar_player_trace_draw("sar_player_trace_draw", "0", "Display the recorded player trace. Requires cheats\n");
+Variable sar_player_trace_draw_through_walls("sar_player_trace_draw_through_walls", "1", "Display the player trace through walls. Requires sar_player_trace_draw");
+Variable sar_player_trace_draw_speed_deltas("sar_player_trace_draw_speed_deltas", "1", "Display the speed deltas. Requires sar_player_trace_draw");
```

Comment:
Missing newlines on cvar descriptions

Reply:
Considering I rewrote basically every help string, that's very funny